autoconfigure/README.md

```diff
@@ -0,0 +1,36 @@
+# Configuration
+
+## Storage configurations
+
+| Configuration | Description | Default |
+|---------------|-------------|---------|
+| `KAFKA_STORE_SPAN_CONSUMER_ENABLED` | Enable partitioning of incoming span batches into individual spans keyed by trace ID. | `true` |
+| `KAFKA_STORE_AGGREGATION_ENABLED` | Enable aggregation spans into traces and then into dependency links. Currently this has to be initiated with span consumer or storage, or initiate manually via `curl /health` | `true` |
+| `KAFKA_STORE_SPAN_STORE_ENABLED` | Enable storage for spans, traces, dependencies, tags, and service names. | `true` |
+| `KAFKA_STORE_BOOTSTRAP_SERVERS` | Kafka bootstrap servers | `localhost:9092` |
+| `KAFKA_STORE_ENSURE_TOPICS` | Ensure topics are created if don't exist. Is recommended to create them manually, to define partition and replication according to your environment. | `true` |
+| `KAFKA_STORE_COMPRESSION_TYPE` | Compression type used to store data in Kafka topics | `NONE` |
```

Comment:
remove this flag for generic kafka properties unless somehow it isn't one.

Reply:
True, I will keep it as a todo to pass overrides of Kafka Stream configs as we have in the Kafka Collector.